Cannot pip wheel turbodbc with numpy 1.8.0 through numpy 1.9.2

I tried to build turbodbc with numpy 1.8.0 using `pip wheel`. 

### Reproducing Steps

On Debian 7:

```
virtualenv venv
pip install numpy==1.8.0
pip wheel turbodbc
```

It seems to work with `numpy==1.10.1` and (I suspect) above.

### Context

gcc version is 4.7.2, python version is 2.7.3

### Error

```
Building wheels for collected packages: turbodbc
  Running setup.py bdist_wheel for turbodbc ... error
  Complete output from command /tmp/turbodbc/venv/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-QP3m6m/turbodbc/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /tmp/tmpaaSi1Ypip-wheel-:
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-x86_64-2.7
  creating build/lib.linux-x86_64-2.7/turbodbc
  copying turbodbc/api_constants.py -> build/lib.linux-x86_64-2.7/turbodbc
  copying turbodbc/connect.py -> build/lib.linux-x86_64-2.7/turbodbc
  copying turbodbc/constructors.py -> build/lib.linux-x86_64-2.7/turbodbc
  copying turbodbc/cursor.py -> build/lib.linux-x86_64-2.7/turbodbc
  copying turbodbc/exceptions.py -> build/lib.linux-x86_64-2.7/turbodbc
  copying turbodbc/data_types.py -> build/lib.linux-x86_64-2.7/turbodbc
  copying turbodbc/connection.py -> build/lib.linux-x86_64-2.7/turbodbc
  copying turbodbc/__init__.py -> build/lib.linux-x86_64-2.7/turbodbc
  running egg_info
  writing turbodbc.egg-info/PKG-INFO
  writing top-level names to turbodbc.egg-info/top_level.txt
  writing dependency_links to turbodbc.egg-info/dependency_links.txt
  reading manifest file 'turbodbc.egg-info/SOURCES.txt'
  reading manifest template 'MANIFEST.in'
  writing manifest file 'turbodbc.egg-info/SOURCES.txt'
  running build_ext
  building 'libturbodbc' extension
  creating build/temp.linux-x86_64-2.7
  creating build/temp.linux-x86_64-2.7/src
  creating build/temp.linux-x86_64-2.7/src/cpp_odbc
  creating build/temp.linux-x86_64-2.7/src/cpp_odbc/level1
  creating build/temp.linux-x86_64-2.7/src/cpp_odbc/level3
  creating build/temp.linux-x86_64-2.7/src/cpp_odbc/level2
  creating build/temp.linux-x86_64-2.7/src/turbodbc
  creating build/temp.linux-x86_64-2.7/src/turbodbc/result_sets
  creating build/temp.linux-x86_64-2.7/src/turbodbc/parameter_sets
  creating build/temp.linux-x86_64-2.7/src/turbodbc/field_translators
  creating build/temp.linux-x86_64-2.7/src/turbodbc/descriptions
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/cpp_odbc/connection.cpp -o build/temp.linux-x86_64-2.7/src/cpp_odbc/connection.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/cpp_odbc/make_environment.cpp -o build/temp.linux-x86_64-2.7/src/cpp_odbc/make_environment.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/cpp_odbc/multi_value_buffer.cpp -o build/temp.linux-x86_64-2.7/src/cpp_odbc/multi_value_buffer.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/cpp_odbc/environment.cpp -o build/temp.linux-x86_64-2.7/src/cpp_odbc/environment.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/cpp_odbc/error.cpp -o build/temp.linux-x86_64-2.7/src/cpp_odbc/error.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/cpp_odbc/statement.cpp -o build/temp.linux-x86_64-2.7/src/cpp_odbc/statement.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/cpp_odbc/cpp_odbc.cpp -o build/temp.linux-x86_64-2.7/src/cpp_odbc/cpp_odbc.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/cpp_odbc/column_description.cpp -o build/temp.linux-x86_64-2.7/src/cpp_odbc/column_description.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/cpp_odbc/level1/level1.cpp -o build/temp.linux-x86_64-2.7/src/cpp_odbc/level1/level1.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/cpp_odbc/level1/api.cpp -o build/temp.linux-x86_64-2.7/src/cpp_odbc/level1/api.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/cpp_odbc/level1/unixodbc_backend_debug.cpp -o build/temp.linux-x86_64-2.7/src/cpp_odbc/level1/unixodbc_backend_debug.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/cpp_odbc/level1/unixodbc_backend.cpp -o build/temp.linux-x86_64-2.7/src/cpp_odbc/level1/unixodbc_backend.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/cpp_odbc/level3/raii_environment.cpp -o build/temp.linux-x86_64-2.7/src/cpp_odbc/level3/raii_environment.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/cpp_odbc/level3/raii_connection.cpp -o build/temp.linux-x86_64-2.7/src/cpp_odbc/level3/raii_connection.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/cpp_odbc/level3/level3.cpp -o build/temp.linux-x86_64-2.7/src/cpp_odbc/level3/level3.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/cpp_odbc/level3/raii_statement.cpp -o build/temp.linux-x86_64-2.7/src/cpp_odbc/level3/raii_statement.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/cpp_odbc/level2/api.cpp -o build/temp.linux-x86_64-2.7/src/cpp_odbc/level2/api.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/cpp_odbc/level2/handles.cpp -o build/temp.linux-x86_64-2.7/src/cpp_odbc/level2/handles.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/cpp_odbc/level2/input_string_buffer.cpp -o build/temp.linux-x86_64-2.7/src/cpp_odbc/level2/input_string_buffer.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/cpp_odbc/level2/level2.cpp -o build/temp.linux-x86_64-2.7/src/cpp_odbc/level2/level2.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/cpp_odbc/level2/string_buffer.cpp -o build/temp.linux-x86_64-2.7/src/cpp_odbc/level2/string_buffer.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/cpp_odbc/level2/level1_connector.cpp -o build/temp.linux-x86_64-2.7/src/cpp_odbc/level2/level1_connector.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/connection.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/connection.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/make_field_translator.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/make_field_translator.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/buffer_size.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/buffer_size.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/command.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/command.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/description.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/description.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/cursor.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/cursor.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/make_description.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/make_description.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/column.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/column.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/connect.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/connect.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/field_translator.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/field_translator.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/parameter.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/parameter.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/result_sets/python_result_set.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/result_sets/python_result_set.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/result_sets/field_result_set.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/result_sets/field_result_set.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/result_sets/double_buffered_result_set.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/result_sets/double_buffered_result_set.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/result_sets/row_based_result_set.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/result_sets/row_based_result_set.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/result_sets/bound_result_set.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/result_sets/bound_result_set.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/result_sets/result_set.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/result_sets/result_set.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/parameter_sets/set_field.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/parameter_sets/set_field.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/parameter_sets/field_parameter_set.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/parameter_sets/field_parameter_set.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/parameter_sets/bound_parameter_set.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/parameter_sets/bound_parameter_set.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/field_translators/timestamp_translator.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/field_translators/timestamp_translator.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/field_translators/string_translator.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/field_translators/string_translator.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/field_translators/int64_translator.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/field_translators/int64_translator.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/field_translators/boolean_translator.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/field_translators/boolean_translator.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/field_translators/date_translator.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/field_translators/date_translator.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/field_translators/float64_translator.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/field_translators/float64_translator.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/descriptions/boolean_description.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/descriptions/boolean_description.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/descriptions/date_description.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/descriptions/date_description.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/descriptions/timestamp_description.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/descriptions/timestamp_description.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/descriptions/floating_point_description.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/descriptions/floating_point_description.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/descriptions/string_description.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/descriptions/string_description.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc/descriptions/integer_description.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc/descriptions/integer_description.o --std=c++11
  g++ -pthread -shared -Wl,-O1 -Wl,-Bsymbolic-functions -Wl,-z,relro build/temp.linux-x86_64-2.7/src/cpp_odbc/connection.o build/temp.linux-x86_64-2.7/src/cpp_odbc/make_environment.o build/temp.linux-x86_64-2.7/src/cpp_odbc/multi_value_buffer.o build/temp.linux-x86_64-2.7/src/cpp_odbc/environment.o build/temp.linux-x86_64-2.7/src/cpp_odbc/error.o build/temp.linux-x86_64-2.7/src/cpp_odbc/statement.o build/temp.linux-x86_64-2.7/src/cpp_odbc/cpp_odbc.o build/temp.linux-x86_64-2.7/src/cpp_odbc/column_description.o build/temp.linux-x86_64-2.7/src/cpp_odbc/level1/level1.o build/temp.linux-x86_64-2.7/src/cpp_odbc/level1/api.o build/temp.linux-x86_64-2.7/src/cpp_odbc/level1/unixodbc_backend_debug.o build/temp.linux-x86_64-2.7/src/cpp_odbc/level1/unixodbc_backend.o build/temp.linux-x86_64-2.7/src/cpp_odbc/level3/raii_environment.o build/temp.linux-x86_64-2.7/src/cpp_odbc/level3/raii_connection.o build/temp.linux-x86_64-2.7/src/cpp_odbc/level3/level3.o build/temp.linux-x86_64-2.7/src/cpp_odbc/level3/raii_statement.o build/temp.linux-x86_64-2.7/src/cpp_odbc/level2/api.o build/temp.linux-x86_64-2.7/src/cpp_odbc/level2/handles.o build/temp.linux-x86_64-2.7/src/cpp_odbc/level2/input_string_buffer.o build/temp.linux-x86_64-2.7/src/cpp_odbc/level2/level2.o build/temp.linux-x86_64-2.7/src/cpp_odbc/level2/string_buffer.o build/temp.linux-x86_64-2.7/src/cpp_odbc/level2/level1_connector.o build/temp.linux-x86_64-2.7/src/turbodbc/connection.o build/temp.linux-x86_64-2.7/src/turbodbc/make_field_translator.o build/temp.linux-x86_64-2.7/src/turbodbc/buffer_size.o build/temp.linux-x86_64-2.7/src/turbodbc/command.o build/temp.linux-x86_64-2.7/src/turbodbc/description.o build/temp.linux-x86_64-2.7/src/turbodbc/cursor.o build/temp.linux-x86_64-2.7/src/turbodbc/make_description.o build/temp.linux-x86_64-2.7/src/turbodbc/column.o build/temp.linux-x86_64-2.7/src/turbodbc/connect.o build/temp.linux-x86_64-2.7/src/turbodbc/field_translator.o build/temp.linux-x86_64-2.7/src/turbodbc/parameter.o build/temp.linux-x86_64-2.7/src/turbodbc/result_sets/python_result_set.o build/temp.linux-x86_64-2.7/src/turbodbc/result_sets/field_result_set.o build/temp.linux-x86_64-2.7/src/turbodbc/result_sets/double_buffered_result_set.o build/temp.linux-x86_64-2.7/src/turbodbc/result_sets/row_based_result_set.o build/temp.linux-x86_64-2.7/src/turbodbc/result_sets/bound_result_set.o build/temp.linux-x86_64-2.7/src/turbodbc/result_sets/result_set.o build/temp.linux-x86_64-2.7/src/turbodbc/parameter_sets/set_field.o build/temp.linux-x86_64-2.7/src/turbodbc/parameter_sets/field_parameter_set.o build/temp.linux-x86_64-2.7/src/turbodbc/parameter_sets/bound_parameter_set.o build/temp.linux-x86_64-2.7/src/turbodbc/field_translators/timestamp_translator.o build/temp.linux-x86_64-2.7/src/turbodbc/field_translators/string_translator.o build/temp.linux-x86_64-2.7/src/turbodbc/field_translators/int64_translator.o build/temp.linux-x86_64-2.7/src/turbodbc/field_translators/boolean_translator.o build/temp.linux-x86_64-2.7/src/turbodbc/field_translators/date_translator.o build/temp.linux-x86_64-2.7/src/turbodbc/field_translators/float64_translator.o build/temp.linux-x86_64-2.7/src/turbodbc/descriptions/boolean_description.o build/temp.linux-x86_64-2.7/src/turbodbc/descriptions/date_description.o build/temp.linux-x86_64-2.7/src/turbodbc/descriptions/timestamp_description.o build/temp.linux-x86_64-2.7/src/turbodbc/descriptions/floating_point_description.o build/temp.linux-x86_64-2.7/src/turbodbc/descriptions/string_description.o build/temp.linux-x86_64-2.7/src/turbodbc/descriptions/integer_description.o -lodbc -lboost_python -o build/lib.linux-x86_64-2.7/libturbodbc.so
  building 'turbodbc_intern' extension
  creating build/temp.linux-x86_64-2.7/src/turbodbc_python
  creating build/temp.linux-x86_64-2.7/src/turbodbc_python/python_bindings
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc_python/determine_parameter_type.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc_python/determine_parameter_type.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc_python/python_parameter_set.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc_python/python_parameter_set.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc_python/python_bindings/connection.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc_python/python_bindings/connection.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc_python/python_bindings/python_result_set.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc_python/python_bindings/python_result_set.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc_python/python_bindings/column_info.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc_python/python_bindings/column_info.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc_python/python_bindings/buffer_size.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc_python/python_bindings/buffer_size.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc_python/python_bindings/unicode.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc_python/python_bindings/unicode.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc_python/python_bindings/error.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc_python/python_bindings/error.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc_python/python_bindings/cursor.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc_python/python_bindings/cursor.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc_python/python_bindings/python_parameter_set.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc_python/python_bindings/python_parameter_set.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc_python/python_bindings/connect.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc_python/python_bindings/connect.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/usr/include/python2.7 -c src/turbodbc_python/python_bindings/module.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc_python/python_bindings/module.o --std=c++11
  g++ -pthread -shared -Wl,-O1 -Wl,-Bsymbolic-functions -Wl,-z,relro build/temp.linux-x86_64-2.7/src/turbodbc_python/determine_parameter_type.o build/temp.linux-x86_64-2.7/src/turbodbc_python/python_parameter_set.o build/temp.linux-x86_64-2.7/src/turbodbc_python/python_bindings/connection.o build/temp.linux-x86_64-2.7/src/turbodbc_python/python_bindings/python_result_set.o build/temp.linux-x86_64-2.7/src/turbodbc_python/python_bindings/column_info.o build/temp.linux-x86_64-2.7/src/turbodbc_python/python_bindings/buffer_size.o build/temp.linux-x86_64-2.7/src/turbodbc_python/python_bindings/unicode.o build/temp.linux-x86_64-2.7/src/turbodbc_python/python_bindings/error.o build/temp.linux-x86_64-2.7/src/turbodbc_python/python_bindings/cursor.o build/temp.linux-x86_64-2.7/src/turbodbc_python/python_bindings/python_parameter_set.o build/temp.linux-x86_64-2.7/src/turbodbc_python/python_bindings/connect.o build/temp.linux-x86_64-2.7/src/turbodbc_python/python_bindings/module.o -Lbuild/lib.linux-x86_64-2.7 -lodbc -lboost_python -lturbodbc -o build/lib.linux-x86_64-2.7/turbodbc_intern.so -Wl,-rpath,$ORIGIN
  building 'turbodbc_numpy_support' extension
  creating build/temp.linux-x86_64-2.7/src/turbodbc_numpy
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include -I/usr/include/python2.7 -c src/turbodbc_numpy/python_bindings.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc_numpy/python_bindings.o --std=c++11
  gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -fPIC -Iinclude/ -I/tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include -I/usr/include/python2.7 -c src/turbodbc_numpy/string_column.cpp -o build/temp.linux-x86_64-2.7/src/turbodbc_numpy/string_column.o --std=c++11
  In file included from /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:4:0,
                   from /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarrayobject.h:17,
                   from include/turbodbc_numpy/ndarrayobject.h:13,
                   from include/turbodbc_numpy/masked_column.h:3,
                   from include/turbodbc_numpy/string_column.h:3,
                   from src/turbodbc_numpy/string_column.cpp:1:
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/npy_common.h:301:2: error: #error Must use Python with unicode enabled.
  In file included from /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:4:0,
                   from /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarrayobject.h:17,
                   from include/turbodbc_numpy/ndarrayobject.h:13,
                   from include/turbodbc_numpy/masked_column.h:3,
                   from include/turbodbc_numpy/string_column.h:3,
                   from src/turbodbc_numpy/string_column.cpp:1:
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/npy_common.h:164:9: error: ‘Py_intptr_t’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/npy_common.h:165:9: error: ‘Py_uintptr_t’ does not name a type
  In file included from /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:4:0,
                   from /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarrayobject.h:17,
                   from include/turbodbc_numpy/ndarrayobject.h:13,
                   from include/turbodbc_numpy/masked_column.h:3,
                   from include/turbodbc_numpy/string_column.h:3,
                   from src/turbodbc_numpy/string_column.cpp:1:
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/npy_common.h:949:9: error: ‘npy_uint16’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/npy_common.h:950:9: error: ‘npy_half’ does not name a type
  In file included from /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarrayobject.h:17:0,
                   from include/turbodbc_numpy/ndarrayobject.h:13,
                   from include/turbodbc_numpy/masked_column.h:3,
                   from include/turbodbc_numpy/string_column.h:3,
                   from src/turbodbc_numpy/string_column.cpp:1:
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:363:9: error: ‘PyObject’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:364:35: error: typedef ‘PyArray_SetItemFunc’ is initialized (use decltype instead)
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:364:35: error: ‘PyObject’ was not declared in this scope
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:364:45: error: expected primary-expression before ‘,’ token
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:364:47: error: expected primary-expression before ‘void’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:364:55: error: expected primary-expression before ‘void’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:366:46: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:366:64: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:367:38: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:379:38: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:379:48: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:381:40: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:381:58: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:382:32: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:384:56: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:391:32: error: typedef ‘PyArray_ScanFunc’ is initialized (use decltype instead)
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:391:32: error: ‘FILE’ was not declared in this scope
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:391:38: error: ‘fp’ was not declared in this scope
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:391:42: error: expected primary-expression before ‘void’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:392:32: error: expected primary-expression before ‘char’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:392:46: error: expected primary-expression before ‘struct’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:396:40: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:398:40: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:399:43: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:399:55: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:400:45: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:400:55: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:401:37: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:401:49: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:403:48: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:403:60: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:403:70: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:404:40: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:404:52: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:407:50: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:411:47: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:413:62: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:414:54: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:415:60: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:416:40: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:416:60: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:417:40: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:417:59: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:421:9: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:440:9: error: ‘PyArray_GetItemFunc’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:441:9: error: ‘PyArray_SetItemFunc’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:473:9: error: ‘PyArray_ScanFunc’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:513:9: error: ‘PyObject’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:573:9: error: ‘PyObject_HEAD’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:609:9: error: ‘PyObject’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:614:9: error: ‘PyObject’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:621:9: error: ‘PyObject’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:632:9: error: ‘PyObject’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:647:5: error: ‘PyObject_HEAD’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:653:5: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:658:5: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:677:5: error: ‘PyObject’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:683:5: error: ‘PyObject’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:699:9: error: ‘PyObject_HEAD’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:707:9: error: ‘PyObject_HEAD’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:715:9: error: ‘PyObject_HEAD’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:718:9: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:738:9: error: ‘npy_int32’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:744:9: error: ‘npy_int32’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:747:53: error: ‘PyObject’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:976:39: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1053:66: error: ‘npy_intp’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1056:9: error: ‘PyObject_HEAD’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1058:9: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1059:9: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1060:9: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1061:9: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1062:9: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1063:9: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1068:9: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1069:9: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1070:9: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1208:9: error: ‘PyObject_HEAD’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1210:9: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1211:9: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1213:9: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1266:9: error: ‘PyObject_HEAD’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1274:9: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1276:9: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1278:9: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1297:9: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1310:9: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1320:9: error: ‘npy_uint32’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1331:9: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1339:9: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1342:9: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1355:5: error: ‘PyObject_HEAD’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1361:5: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1362:5: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1363:5: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1364:5: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1365:5: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1366:5: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1371:5: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1372:5: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1373:5: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1379:5: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1382:5: error: ‘npy_intp’ does not name a type
  In file included from /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1417:0,
                   from /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarrayobject.h:17,
                   from include/turbodbc_numpy/ndarrayobject.h:13,
                   from include/turbodbc_numpy/masked_column.h:3,
                   from include/turbodbc_numpy/string_column.h:3,
                   from src/turbodbc_numpy/string_column.cpp:1:
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h: In function ‘int _PyArrayNeighborhoodIter_IncrCoord(PyArrayNeighborhoodIterObject*)’:
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:40:5: error: ‘npy_intp’ was not declared in this scope
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:40:14: error: expected ‘;’ before ‘i’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:42:10: error: ‘i’ was not declared in this scope
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:42:20: error: ‘struct PyArrayNeighborhoodIterObject’ has no member named ‘nd’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:43:9: error: ‘wb’ was not declared in this scope
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:43:9: error: ‘struct PyArrayNeighborhoodIterObject’ has no member named ‘coordinates’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:43:9: error: ‘struct PyArrayNeighborhoodIterObject’ has no member named ‘bounds’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:43:9: error: ‘struct PyArrayNeighborhoodIterObject’ has no member named ‘coordinates’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:43:9: error: ‘struct PyArrayNeighborhoodIterObject’ has no member named ‘coordinates’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:43:9: error: ‘struct PyArrayNeighborhoodIterObject’ has no member named ‘bounds’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h: In function ‘int _PyArrayNeighborhoodIter_IncrCoord2D(PyArrayNeighborhoodIterObject*)’:
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:55:5: error: ‘npy_intp’ was not declared in this scope
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:55:14: error: expected ‘;’ before ‘wb’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:57:5: error: ‘wb’ was not declared in this scope
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:57:5: error: ‘struct PyArrayNeighborhoodIterObject’ has no member named ‘coordinates’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:57:5: error: ‘struct PyArrayNeighborhoodIterObject’ has no member named ‘bounds’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:57:5: error: ‘struct PyArrayNeighborhoodIterObject’ has no member named ‘coordinates’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:57:5: error: ‘struct PyArrayNeighborhoodIterObject’ has no member named ‘coordinates’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:57:5: error: ‘struct PyArrayNeighborhoodIterObject’ has no member named ‘bounds’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:58:5: error: ‘struct PyArrayNeighborhoodIterObject’ has no member named ‘coordinates’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:58:5: error: ‘struct PyArrayNeighborhoodIterObject’ has no member named ‘bounds’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:58:5: error: ‘struct PyArrayNeighborhoodIterObject’ has no member named ‘coordinates’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:58:5: error: ‘struct PyArrayNeighborhoodIterObject’ has no member named ‘coordinates’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:58:5: error: ‘struct PyArrayNeighborhoodIterObject’ has no member named ‘bounds’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h: In function ‘int PyArrayNeighborhoodIter_Next(PyArrayNeighborhoodIterObject*)’:
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:71:69: error: ‘struct PyArrayNeighborhoodIterObject’ has no member named ‘coordinates’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h: In function ‘int PyArrayNeighborhoodIter_Reset(PyArrayNeighborhoodIterObject*)’:
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:82:5: error: ‘npy_intp’ was not declared in this scope
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:82:14: error: expected ‘;’ before ‘i’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:84:10: error: ‘i’ was not declared in this scope
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:84:27: error: ‘struct PyArrayNeighborhoodIterObject’ has no member named ‘nd’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:85:15: error: ‘struct PyArrayNeighborhoodIterObject’ has no member named ‘coordinates’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:85:38: error: ‘struct PyArrayNeighborhoodIterObject’ has no member named ‘bounds’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h:87:69: error: ‘struct PyArrayNeighborhoodIterObject’ has no member named ‘coordinates’
  In file included from /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarrayobject.h:17:0,
                   from include/turbodbc_numpy/ndarrayobject.h:13,
                   from include/turbodbc_numpy/masked_column.h:3,
                   from include/turbodbc_numpy/string_column.h:3,
                   from src/turbodbc_numpy/string_column.cpp:1:
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h: In function ‘void* PyArray_DATA(PyArrayObject*)’:
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1462:43: error: ‘PyArrayObject_fields’ has no member named ‘data’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h: In function ‘char* PyArray_BYTES(PyArrayObject*)’:
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1468:43: error: ‘PyArrayObject_fields’ has no member named ‘data’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h: At global scope:
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1471:19: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1477:19: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1483:19: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1489:19: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1495:44: error: ‘PyObject’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1513:19: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1531:19: error: ‘PyObject’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1539:52: error: ‘PyObject’ has not been declared
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h: In function ‘int PyArray_SETITEM(PyArrayObject*, char*, int*)’:
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1541:53: error: ‘struct PyArray_ArrFuncs’ has no member named ‘setitem’
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h: At global scope:
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1580:19: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1739:5: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1772:5: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1777:5: error: ‘npy_intp’ does not name a type
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1781:5: error: ‘PyObject’ does not name a type
  In file included from include/turbodbc_numpy/ndarrayobject.h:13:0,
                   from include/turbodbc_numpy/masked_column.h:3,
                   from include/turbodbc_numpy/string_column.h:3,
                   from src/turbodbc_numpy/string_column.cpp:1:
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarrayobject.h: In function ‘void PyArray_XDECREF_ERR(PyArrayObject*)’:
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarrayobject.h:166:16: error: ‘NULL’ was not declared in this scope
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarrayobject.h:168:68: error: ‘PyArray_BASE’ was not declared in this scope
  /tmp/turbodbc/venv/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarrayobject.h:172:22: error: ‘Py_DECREF’ was not declared in this scope
  error: command 'gcc' failed with exit status 1
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -20,7 +20,7 @@ before_script:
   - sudo -u postgres psql -U postgres -c "ALTER USER postgres WITH PASSWORD 'password';"
   - mysql -u root -e 'CREATE DATABASE test_db;'
 script:
-  - sudo pip install mock numpy
+  - sudo pip install mock numpy==1.8.0
   - export ODBCSYSINI=${PWD}/travis/odbc
   - mkdir build && cd build && cmake -DCMAKE_INSTALL_PREFIX=./dist .. 
   - make -j4

--- a/cpp/turbodbc_numpy/Library/turbodbc_numpy/ndarrayobject.h
+++ b/cpp/turbodbc_numpy/Library/turbodbc_numpy/ndarrayobject.h
@@ -10,4 +10,5 @@
 // as to why these defines are necessary
 #define PY_ARRAY_UNIQUE_SYMBOL turbodbc_numpy_API
 #define NO_IMPORT_ARRAY
+#include <Python.h>
 #include <numpy/ndarrayobject.h>

